site_root/input/resources/examples/Observation-vs-001-no-phi.json

```diff
@@ -0,0 +1,41 @@
+{
+  "resourceType": "Observation",
+  "id": "vs-001-no-phi",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/kfdrc-vital-status"
```

Comment:
I was about to say it should point to a `kfdrc-vital-status-no-phi` profile (which you would have to create), BUT I think it would end up being the same as the `kfdrc-phenotype-no-phi` since all that profile does is remove all of the PHI fields on the base `Observation` resource. I think that's exactly what you want so you should just be able to change:  `http://fhir.kids-first.io/StructureDefinition/kfdrc-vital-status` to `http://fhir.kids-first.io/StructureDefinition/kfdrc-phenotype-no-phi`. 

In a separate PR we should probably rename `kfdrc-phenotype-no-phi` to  something more generic like `observation-no-phi`.

Reply:
@znatty22 Done! Let me know if it looks good.